.github/workflows/azure-static-webapp.yml

```diff
@@ -0,0 +1,79 @@
+name: Azure Static Web Apps CI/CD
+
+on:
+  push:
+    branches:
+      - master
+      - release/*
+  pull_request:
+    types: [opened, synchronize, reopened, closed]
+    branches:
+      - master
+      - release/*
+
+jobs:
+  build_and_deploy_job:
+    env:
+      DIST_PATH: Uno.Gallery/Uno.Gallery.Wasm/bin/Release/net5.0/dist
+
+    if: github.event_name == 'push' || (github.event_name == 'pull_request' && github.event.action != 'closed')
+    runs-on: ubuntu-latest
+    container: 'unoplatform/wasm-build:3.0'
+    name: Build and Deploy Job
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # Required for gitversion
+          submodules: true
+
+      - name: Install GitVersion
+        uses: gittools/actions/gitversion/setup@v0.9.7
+        with:
+          versionSpec: '5.x'
+
+      - name: Determine Version
+        id: gitversion
+        uses: gittools/actions/gitversion/execute@v0.9.7
+        with:
+          useConfigFile: true
+          configFilePath: build/gitversion.yml
+
+      - name: Setup dotnet 5.0.x
+        uses: actions/setup-dotnet@v1.7.2
+        with:
+          dotnet-version: '5.0.x'
+
+      - run: |
+          cd Uno.Gallery/Uno.Gallery.Wasm
+          dotnet build -c Release "/p:PackageVersion=${{ steps.gitversion.outputs.fullSemVer }}"
+      - uses: actions/upload-artifact@v2
+        with:
+          name: wasm-site
+          path: ${{ env.DIST_PATH }}
+
+      - name: Publish to Azure Static WebApps
+        if: ${{ github.ref != 'refs/heads/release/prod' }}
+        id: builddeploy_uno
+        uses: Azure/static-web-apps-deploy@v0.0.1-preview
+        with:
+          azure_static_web_apps_api_token: ${{ secrets.AZURE_STATIC_WEB_APPS_API_TOKEN_STAGING }}
+          repo_token: ${{ secrets.GITHUB_TOKEN }} # Used for Github integrations (i.e. PR comments)
+          action: "upload"
+          ###### Repository/Build Configurations - These values can be configured to match your app requirements. ######
+          # For more information regarding Static Web App workflow configurations, please visit: https://aka.ms/swaworkflowconfig
+          app_location: "${{ env.DIST_PATH }}" # App source code path
+          api_location: "" # Api source code path - optional
+          output_location: "" # Built app content directory - optional
+          ###### End of Repository/Build Configurations ######
+
+  close_pull_request_job:
+    if: github.event_name == 'pull_request' && github.event.action == 'closed'
+    runs-on: ubuntu-latest
+    name: Close Pull Request Job
+    steps:
+      - name: Close Pull Request
+        id: closepullrequest
+        uses: Azure/static-web-apps-deploy@v0.0.1-preview
+        with:
+          azure_static_web_apps_api_token: ${{ secrets.AZURE_STATIC_WEB_APPS_API_TOKEN_THANKFUL_MOSS_05F00D00F }}
```

Comment:
```suggestion
          azure_static_web_apps_api_token: ${{ secrets.AZURE_STATIC_WEB_APPS_API_TOKEN_STAGING }}
```

Reply:
You copied it from somewhere that's likely incorrect (the token name is not the right one). We'll need to fix that one as well.

Reply:
@jeromelaban PR available here : https://github.com/unoplatform/Uno.SQLitePCLRaw.Wasm/pull/27